internal/x/cmd/cmd.go

```diff
@@ -498,3 +506,7 @@ func (f *flags) bindKeepaliveTime(flagSet *pflag.FlagSet) {
 func (f *flags) bindUncomment(flagSet *pflag.FlagSet) {
 	flagSet.BoolVar(&f.uncomment, "uncomment", false, "Uncomment the example config settings.")
 }
+
+func (f *flags) bindHarbormasterOutput(flagSet *pflag.FlagSet) {
+	flagSet.BoolVar(&f.harbormasterOutput, "harbormaster-output", false, "Print failures in JSON compatible with the Harbormaster API.")
```

Comment:
maybe just `harbormaster`? @amckinney thoughts?

Reply:
That was my first thought, happy to change to that.

Reply:
Yea, let's shorten to `harbormaster`.

Reply:
Done